sig-security-assessments/Documentation/recipe-book.md

```diff
@@ -0,0 +1,52 @@
+# Recipe Book 👩‍🍳
+🎉Yay! You have decided to do a Self-assessment and make your corner of Kubernetes more secure! We are so happy 
+to have you here. Below is an outline of the journey to get to a successful Self-assessment.
+
+## Preparation
+Set yourself up for success – build your team and get organized
+1.	Figure out who can do this Self-assessment with you. You will need a security expert and a counterpart who owns the 
+project and is an expert in it. If you are requesting a Security Self-assessment you can reach out to the team in
+[#sig-security](https://kubernetes.slack.com/messages/sig-security) to see if there is someone interested in helping to
+champion the assessment.
```

Comment:
If we are allowing this to be a bit more self serve should we have a dedicated sub-project channel for folks to easily find potential security champions for their assessments? 

We could also have an intake form so project owners could make a formal request to start a Self-assessment?

Reply:
There is a dedicated Self Assessment channel already that I think we can totally use to source security champions: https://kubernetes.slack.com/archives/C0441E11REC. Do you think that's sufficient?

The intake form lives here when you add an issue to sig-security: https://github.com/kubernetes/sig-security/issues/new/choose